<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Farm_page</class>
 <widget class="QMainWindow" name="Farm_page">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1542</width>
    <height>1141</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="sheep_butt">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>370</y>
      <width>131</width>
      <height>91</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="cowp_butt">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>500</y>
      <width>271</width>
      <height>161</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="henp_butt">
    <property name="geometry">
     <rect>
      <x>1240</x>
      <y>420</y>
      <width>261</width>
      <height>111</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="henl">
    <property name="geometry">
     <rect>
      <x>1230</x>
      <y>420</y>
      <width>131</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="silo_butt">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>140</y>
      <width>151</width>
      <height>291</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="shop_butt">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>180</y>
      <width>201</width>
      <height>161</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="car_l">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>111</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="cow_l">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>480</y>
      <width>211</width>
      <height>171</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="cow_l_2">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>530</y>
      <width>151</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="silo_l">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>110</y>
      <width>331</width>
      <height>351</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="cowp_l">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>430</y>
      <width>451</width>
      <height>341</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="storge_butt">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>270</y>
      <width>261</width>
      <height>151</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="henp_l">
    <property name="geometry">
     <rect>
      <x>1290</x>
      <y>390</y>
      <width>211</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="gandom_butt">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>470</y>
      <width>131</width>
      <height>131</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="gandom_l">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>270</y>
      <width>401</width>
      <height>541</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="levelpix_l">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>0</y>
      <width>341</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QProgressBar" name="exp_progressBar">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>10</y>
      <width>261</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QProgressBar::chunk {background-color:  #ffbe3d;


}
 QProgressBar{text-align: center;
 border: 2px solid grey;
   border-radius: 20px;
}
</string>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLabel" name="cointakjte_l">
    <property name="geometry">
     <rect>
      <x>1380</x>
      <y>40</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="cointakjte_l_2">
    <property name="geometry">
     <rect>
      <x>1460</x>
      <y>40</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="coin_n_l">
    <property name="geometry">
     <rect>
      <x>1390</x>
      <y>40</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Jokerman</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string>20</string>
    </property>
   </widget>
   <widget class="QLabel" name="yonje_l">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>520</y>
      <width>401</width>
      <height>271</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="yonje_butt">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>610</y>
      <width>131</width>
      <height>151</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="level_n_l">
    <property name="geometry">
     <rect>
      <x>1220</x>
      <y>0</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Jokerman</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
color: rgb(255, 217, 25);</string>
    </property>
    <property name="text">
     <string>20</string>
    </property>
   </widget>
   <zorder>gandom_l</zorder>
   <zorder>henp_l</zorder>
   <zorder>cowp_l</zorder>
   <zorder>silo_l</zorder>
   <zorder>sheep_butt</zorder>
   <zorder>henl</zorder>
   <zorder>silo_butt</zorder>
   <zorder>shop_butt</zorder>
   <zorder>car_l</zorder>
   <zorder>cow_l</zorder>
   <zorder>cow_l_2</zorder>
   <zorder>cowp_butt</zorder>
   <zorder>storge_butt</zorder>
   <zorder>henp_butt</zorder>
   <zorder>levelpix_l</zorder>
   <zorder>exp_progressBar</zorder>
   <zorder>cointakjte_l</zorder>
   <zorder>cointakjte_l_2</zorder>
   <zorder>coin_n_l</zorder>
   <zorder>yonje_l</zorder>
   <zorder>yonje_butt</zorder>
   <zorder>gandom_butt</zorder>
   <zorder>level_n_l</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1542</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
